src/sass/_Fabric.Typography.Mixins.scss

```diff
@@ -5,7 +5,7 @@
 // --------------------------------------------------
 // Fabric Core Typography mixins
 
-@mixin ms-baseFont() {
+@mixin ms-baseFont {
   font-family: $ms-font-family-base;
```

Comment:
It just seemed like a reasonable base property to set along with color. We can leave it out and discuss more if you'd like.